docs/how-to-work-on-coding-challenges.md

```diff
@@ -231,7 +225,7 @@ Proper nouns should use correct capitalization when possible. Below is a list of
 
 - JavaScript (capital letters in "J" and "S" and no abbreviations)
 - Node.js
-- Front-end development (adjective form with a dash) is when you're working on the front end (noun form with no dash). The same goes with "back end", "full stack", and many other compound terms.
+- Although sometimes inaccurate, non-hyphenated forms of 'back end' and 'front end' should be used, as they are more widely used.
```

Comment:
Again...Prettier except this line

Let me know, if I should undo the prettifying.

Reply:
At some point we should run prettier on all of the docs.